lib/order.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :dishes, :order, :menu
+
+  def initialize
+    @menu = Menu::MENU
```

Comment:
Here we're implicitly relying on a constant in the Menu class. Instead, we could make this dependency explicit and use [dependency injection](https://github.com/makersacademy/skills-workshops/blob/f5b4801840fe07d26ff70341652dc81dcda12289/practicals/object_oriented_design/dependency_injection.md) which will allow us to decouple the classes and isolate the unit tests by using mocking